Makefile.configure

```diff
@@ -2,15 +2,15 @@ ifeq ($(PYTHON_VER),2.7)
 	PYTHON_CONFIGURE_ARGS ?=
 endif
 ifeq ($(PYTHON_VER),3.5)
-	PYTHON_CONFIGURE_ARGS ?= --without-ensurepip
+	PYTHON_CONFIGURE_ARGS ?=
 endif
 ifeq ($(PYTHON_VER),3.6)
-	PYTHON_CONFIGURE_ARGS ?= --without-ensurepip
+	PYTHON_CONFIGURE_ARGS ?=
 endif
 ifeq ($(PYTHON_VER),3.7)
-	PYTHON_CONFIGURE_ARGS ?= --without-ensurepip
+	PYTHON_CONFIGURE_ARGS ?=
 endif
 ifeq ($(PYTHON_VER),3.8)
-	PYTHON_CONFIGURE_ARGS ?= --without-ensurepip
+	PYTHON_CONFIGURE_ARGS ?=
 endif
```

Comment:
Maybe simplify the entire series of python version checks into a single unconditional

```
PYTHON_CONFIGURE_ARGS ?=
```

Reply:
And yup again